core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -196,11 +205,45 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public void showSubtitleTrack(PlayerSubtitleTrack subtitleTrack) {
+        setSubtitleRenderer();
+        exoPlayer.selectSubtitleTrack(subtitleTrack);
+        playerView.showSubtitles();
+    }
+
+    @Override
+    public void showFirstAvailableSubtitleTrack() {
+        setSubtitleRenderer();
+        exoPlayer.selectFirstAvailableSubtitlesTrack();
+        playerView.showSubtitles();
+    }
+
+    private void setSubtitleRenderer() {
+        exoPlayer.setSubtitleRendererOutput(new TextRenderer.Output() {
+            @Override
+            public void onCues(List<Cue> list) {
+                playerView.setSubtitleCue(list);
+            }
+        });
+    }
+
+    @Override
+    public void hideSubtitleTrack() {
```

Comment:
We should probably also unset the `SubtitleRendererOutput` as it is set in `showSubtitleTrack`